File: 프로그래머스/0/181886. 5명씩/5명씩.js
function solution(names) {
    const list = [];
    // 5명 그룹의 선두 인원의 이름을 받을 list를 만든다
    for (let i = 0; i < names.length; i += 5) {
        list.push(names[i]);
    } // 0번째 부터 그 뒤 5번째씩의 사람을 list에 추가한다
    return list;
}

File: 프로그래머스/unrated/181886. 5명씩/5명씩.java
import java.util.*;

class Solution {
    public String[] solution(String[] names) {
        List<String> list = new ArrayList<>();
        // 5명 그룹의 선두 인원의 이름을 받을 list를 만든다
        for (int i = 0; i < names.length; i+=5) {
          list.add(names[i]);
        } // 0번째 부터 그 뒤 5번째씩의 사람을 list에 추가한다
        
        String[] answer = list.toArray(new String[0]);
        // list를 배열로 변환해준다
        return answer;
    }
}